docusign/connector.go

```diff
@@ -28,19 +31,15 @@ func NewConnector(opts ...Option) (conn *Connector, outErr error) {
 		},
 	}
 
-	// Read provider info
-	conn.ProviderInfo, err = providers.ReadInfo(providers.Docusign)
```

Comment:
Provider information has unresolved {{.server}} variable.
This is handled at later time when `PostAuthInfo` is set.